resource/specs/dev.toml

```diff
@@ -6,6 +6,9 @@ parent_hash = "0x000000000000000000000000000000000000000000000000000000000000000
 timestamp = 0
 difficulty = "0x100"
 uncles_hash = "0x0000000000000000000000000000000000000000000000000000000000000000"
+# run `cargo run cli hashes -b` to get the genesis hash
+hash = "0xd65a017201803f37d87130a94f9e19abd23369f77ee76a369b4e5040c9a8f50d"
```

Comment:
A rebase conflict. We have removed `hash` from `dev.toml`, since updating is tedious for development. `issued_cells` can be deleted as well.